cadquery/occ_impl/assembly.py

```diff
@@ -236,7 +239,10 @@ def _toCAF(el, ancestor, color) -> TDF_Label:
 
         if ancestor:
             # add the current subassy to the higher level assy
-            tool.AddComponent(ancestor, subassy, el.loc.wrapped)
+            if ancestor.Tag() == 1:
```

Comment:
Could you explain in a comment what `Tag() == 1` does/means?

Reply:
I want to apply the top Location to any immediate child subassy of top.  When `Tag() == 1` the ancestor is the top.

I understand in general Tag is not unique in a document.  Comparing Tag was a convenient way to identify the ancestor and when used in this way I did not find any case where it failed.

An alternative would be to compare the ancestor name with the assembly name `assy.name`.